### ISSUE TITLE
Treatment of dosimetry libraries must be changed

### ISSUE THREAD

**Issue by AlbertoBittes** [JADE-V-V/JADE#186]
The current way the code deals with the presence of dosimetry libraries for reaction rates is not rigorous and should be changed.

**Comment by dodu94** [comment]
Please Alberto give more details about the issue

**Comment by AlbertoBittes** [comment]
When the object Test is about to be initialized in computational.py, to avoid the translation of dosimetry libraries, the following lines of code are used.

            if testname in ['FNG Bulk Blanket and Shielding Experiment', 
                            'FNG Tungsten', 
                            'ASPIS Iron-88 benchmark']:
                var = {'00c': lib, '34y': '34y'}
            else:
                var = lib

In this way, the test translates the input by using the D1S libraries logic. However, it is not very rigorous and flexible:
1. Each benchmark using dosimetry libraries should be added to the list every time
2. The non-dosimetry nuclide libraries of the original input must be set as 00c, or the translation won't work correctly
3. To change the suffix dosimetry library for the translation one should change the source code
4. If the library 34y is not in the xsdir I think that the whole translation fails.
5. Doesn't account for the case in which dosimetry libraries are not available, setting a 'default' library
6. In the future we could want to implement comparison between dosimetry libraries results, this is not currently possible
In brief, this works only for our internal xsdir configuration, which is of course something we don't want

**Comment by dodu94** [comment]
This issue should be prioritized now since the behaviour with different codes is most likely broken

**Comment by dodu94** [comment]
I have to check that I implemented this correctly in JADE v4, most likely not. One way to approach this would have to add an optional argument to the Library object to specify if it is a reaction rate lib or not and then add a check ensuring that it is not translated. A bit tricky but can be done. This though made me wonder: how would this be implemented in OpenMC @alexvalentine94 @sbradnam? As far as I know in OpenMC you just point to a cross_section.xml and you cannot specify different libraries for different materials

**Comment by dodu94** [comment]
Found this from Paul: https://openmc.discourse.group/t/using-irdff-ii-cross-section-data-in-openmc/1950/3

**Comment by dodu94** [comment]
Basically, using the EnergyFilter in OpenMC it is possible to obtain the correct reaction rate that uses the correct dosimetry library without having to deal with material translations. This means that to solve this issue:

- [ ] It should be discussed in the documentation how to implement benchmarks with dosimeters in MCNP and OpenMC
- [ ] All benchmarks that have dosimeters and have already been implemented in OpenMC must be updated (if any)
- [ ] A solution to treat dosimetry libraries in MCNP in a general way should be implemented.

It can be assumed that once a dosimetry library has been chosen for a benchmark it cannot be changed as it is considered part of the modelling. Changing it would require a version change of the input itself

**Comment by alexvalentine94** [comment]
Yes I was looking how they were doing it here as an example https://github.com/eepeterson/openmc_fusion_benchmarks/blob/master/models/fng_w/openmc_model.py 

we will need to implement this

**Comment by dodu94** [comment]
MCNP bugged behaviour has been fixed, for OpenMC the filters need to be applied directly in the benchmark input
